style.css

```diff
@@ -20,6 +20,12 @@ body {
     overflow: hidden;
 }
 
+audio {
+    width: 240px;
+    height: 32px;
+    margin: 4px;
+}
+
```

Comment:
@HarryPdot Whenever we deal with pixels, keep the measurements to 4s. i.e. 4px 8px 12px ... 64px 320px etc